index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title> Alexis/Portfolio </title>
+    <title> Portfolio </title>
```

Comment:
- [ ]  I noticed that you change your page title, at the moment, "portfolio" is too generic, titles should adequately and briefly describe the content of the page. Kindly change your title and add the change to the report. For example:
  - Poor titles:
    - Microverse project | Your portfolio.
    - Portifolio project
  - Better page titles:
    - Mark's portfolio
    - Mark full-stack web developer